jekyll/_cci2/insights-glossary.md

```diff
@@ -25,15 +25,16 @@ General metrics appear across the Insights experience and can refer to different
-| Total Duration        | The sum of execution times for all workflows in a project.                                                                                                                                                                                                                                                             |
-| P95 Duration          | The 95th percentile of execution times for an entity in a selected time window (i.e. 95% of runs completed in the same or less amount of this time). <br/> _The 95th percentile is a standard measure used to interpret performance data. It provides a measure of max value when outlier or transient values are excluded._ |
-| P50 (median) Duration | The median execution time. <br/> _Medians are a better measure of central tendency than arithmetic means because they are more resilient to_ skewness _in datasets_.                                                                                                                                                         |
-| Total Credits         | The sum of credits consumed during execution.                                                                                                                                                                                                                                                                          |
-| Success Rate          | The percentage of runs that completed successfully, calculated by `100 x (Successful Runs / All Runs)`                                                                                                                                                                                                                   |
+| Term                                                 | Definition                                                                                                                                                                                                                                                                                                             |
+|------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| {::nomarkdown}<div id="insights-runs">Runs</div>{:/} | The count of executions in a selected time window.                                                                                                                                                                                                                                                                     |
```

Comment:
I tried to just add {: #insights-runs} but it did not work. If I don't add in `{::nomarkdown}` then the `div` is interpreted just as plain text